frontend/app/page.tsx

```
@@ -320,59 +255,44 @@ export default function Home() {
         >
           {/* Section divider */}
           <div className="flex items-center gap-4 mb-12">
-            <div className="h-px flex-1 bg-gradient-to-r from-transparent to-[#58c4dd]/20" />
-            <span className="text-[#58c4dd]/50 text-sm font-mono">// HOW IT WORKS</span>
-            <div className="h-px flex-1 bg-gradient-to-l from-transparent to-[#58c4dd]/20" />
+            <div className="h-px flex-1 bg-gradient-to-r from-transparent to-white/[0.06]" />
+            <span className="text-white/25 text-sm font-mono">// HOW IT WORKS</span>
+            <div className="h-px flex-1 bg-gradient-to-l from-transparent to-white/[0.06]" />
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)